va/caa_test.go

```diff
@@ -179,7 +179,7 @@ func (mock caaMockDNS) LookupCAA(_ context.Context, domain string) ([]*dns.CAA,
 		record.Value = "letsencrypt.org"
 		results = append(results, &record)
 	}
-	return results, nil
+	return results, "", nil
```

Comment:
Would be good to have this return a non-empty string (which could be set on a case-by-case basis above, or derived from something in `record`), so the expectations below have a non-empty `Response=` as well

Reply:
Good catch here. I've gone ahead and populated `response` with `foo` if length of `results` is greater-than `0`